packages/mongo-nav/src/MongoNav.tsx

```diff
@@ -0,0 +1,155 @@
+        invitations:
+          urls?.mongoMenu?.cloud?.invitations ??
+          `${cloudHost}/v2#/preferences/invitations`,
+        tfa: urls?.mongoMenu?.cloud?.tfa ?? `${cloudHost}/v2#/preferences/2fa`,
+      },
+      university: {
+        videoPreferences:
+          urls?.mongoMenu?.university?.videoPreferences ?? `${universityHost}`,
+      },
+      support: {
+        userPreferences:
+          urls?.mongoMenu?.support?.userPreferences ?? `${supportHost}/profile`,
+      },
+      account: {
+        accountURL:
+          urls?.mongoMenu?.account?.accountURL ??
+          `${accountHost}/account/profile/overview`,
+      },
+    },
+    mongoSelect: {
+      viewAllProjects:
+        urls?.mongoSelect?.viewAllProjects ??
+        `${cloudHost}/v2#/org/${data.currentProject.orgId}/projects`,
+      viewAllOrganizations:
+        urls?.mongoSelect?.viewAllOrganizations ??
+        `${cloudHost}/v2#/preferences/organizations`,
+      newProject:
+        urls?.mongoSelect?.newProject ??
+        `${cloudHost}/v2#/org/${data.currentProject.orgId}/projects/create`,
+      orgSettings:
+        urls?.mongoSelect?.newProject ??
+        `${cloudHost}/v2#/org/${data.currentOrganization.orgId}/settings/general`,
+    },
+    orgNav: {
+      settings:
+        urls?.orgNav?.settings ??
+        `${cloudHost}/v2#/org/${data.currentOrganization.orgId}/settings/general`,
+      accessManager:
+        urls?.orgNav?.accessManager ??
+        `${cloudHost}/v2#/org/${data.currentOrganization.orgId}/access/users`,
+      support:
+        urls?.orgNav?.support ??
+        `${cloudHost}/v2#/org/${data.currentOrganization.orgId}/support`,
+      billing:
+        urls?.orgNav?.billing ??
+        `${cloudHost}/v2#/org/${data.currentOrganization.orgId}/billing/overview`,
+      allClusters: urls?.orgNav?.allClusters ?? `${cloudHost}/v2#/clusters`,
+      admin:
+        urls?.orgNav?.admin ?? `${cloudHost}/v2/admin#general/overview/servers`,
+    },
+    projectNav: {
+      settings:
+        urls?.projectNav?.settings ??
+        `${cloudHost}/v2/${data.currentProject.projectId}#settings/groupSettings`,
+      accessManager:
+        urls?.projectNav?.accessManager ??
+        `${cloudHost}/v2/${data.currentProject.projectId}#access`,
+      support:
+        urls?.projectNav?.support ??
+        `${cloudHost}/v2/${data.currentProject.projectId}#info/support`,
+      integrations:
+        urls?.projectNav?.integrations ??
+        `${cloudHost}/v2/${data.currentProject.projectId}#integrations`,
+      alerts:
+        urls?.projectNav?.alerts ??
+        `${cloudHost}/v2/${data.currentProject.projectId}#alerts`,
+      activityFeed:
+        urls?.projectNav?.activityFeed ??
+        `${cloudHost}/v2/${data.currentProject.projectId}#activity`,
+    },
+  };
+
+  return (
+    <>
+      <OrgNav
+        account={data.account}
+        activeProduct={activeProduct}
+        current={data.currentOrganization}
```

Comment:
Ya might consider destructuring `data` to clean up stuff like this a tad.

Reply:
👍